modules/enabling-multi-cluster-console.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * assemblies/web-console.adoc
+
+:_content-type: PROCEDURE
+[id="enable-multi-cluster-console_{context}"]
+= Enabling multi-cluster in the web console
```

Comment:
Should add in a description of what multi-cluster does and why you might want to enable it?